server/static/jsx/scholarship.jsx

```diff
@@ -0,0 +1,96 @@
+/** @jsx React.DOM */
+define(['ext/jquery', 'ext/react'], function($, React) {
+  var ScholarshipContainer = React.createClass({
+    render: function() {
+      var scholarships = this.props.scholarshipData.map(function(data) {
+        return <ScholarshipBox data={data} />;
+      });
+
+      return (
+        <div>
+          <h1> Scholarships </h1>
+          <div className="scholarship-container">{scholarships}</div>
+        </div>
+      );
+    }
+  });
+
+  var ScholarshipBox = React.createClass({
+    getInitialState: function() {
+      return {
+        expanded: false,
+        removed: false
+      }
+    },
+
+    toggleExpansion: function() {
+      this.setState({expanded: !this.state.expanded});
+    },
+
+    removeFromProfile: function() {
+      this.setState({removed: true});
+      $.ajax({
+        type: 'PUT',
+        url: '/api/v1/user/remove_scholarship_from_profile',
+        data: {
+          scholarship_id: this.props.data.id
+        }
+      });
+    },
+
+    render: function() {
+      if (this.state.removed) {
+        return null;
+      }
+
+      var cx = React.addons.classSet;
+      var classes = cx({
+        'scholarship-content': true,
+        'expanded': this.state.expanded
+      });
+
+      scholarshipInner = null;
+
+      if (this.state.expanded) {
+        scholarshipInner = (
+          <div className="scholarship-inner row-fluid">
```

Comment:
nit: Can we make this another React class? One of the advantages of React is it makes composing really easy, so let's use that. :)